inttest/ap-ha3x3/ha3x3_test.go

```diff
@@ -117,26 +117,17 @@ spec:
   timestamp: now
   commands:
     - k0supdate:
-        version: v0.0.0
-        forceupdate: true
+        version: ` + s.K0sUpdateVersion + `
         platforms:
           linux-amd64:
-            url: http://localhost/dist/k0s
+            url: http://localhost/dist/k0s-new
         targets:
           controllers:
             discovery:
-              static:
-                nodes:
-                  - controller0
-                  - controller1
-                  - controller2
+              selector: {}
```

Comment:
Does this need to be changed to selectors? The existing plan was intentionally extremely static, and now this becomes sensitive to the creation of the footloose cluster.

Reply:
Sorry, will change it back to static